example-scripts/module.rhai

```diff
@@ -0,0 +1,5 @@
+//! This script imports an external script as a module.
```

Comment:
This script won't run, as it loads external modules (not supported by WASM).  It is deliberately omitted.